### ISSUE TITLE
unknown capability "CAP_PERFMON"

### ISSUE THREAD

**Issue by oliverchang** [google/gvisor#8529]
### Description

Running Docker with runsc as the runtime, we are running into issues with creating privileged containers:

 

### Steps to reproduce

```
docker run -ti --rm --privileged ubuntu /bin/bash
docker: Error response from daemon: failed to create shim task: OCI runtime create failed: creating container: cannot create sandbox: cannot read client sync file: waiting for sandbox to start: EOF: unknown.
```

### runsc version

```shell
runsc version release-20230130.0
spec: 1.0.2-dev
```

### docker version (if using docker)

```shell
Client: Docker Engine - Community
 Version:           20.10.23
 API version:       1.41
 Go version:        go1.18.10
 Git commit:        7155243
 Built:             Thu Jan 19 17:34:12 2023
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          23.0.1
  API version:      1.42 (minimum version 1.12)
  Go version:       go1.19.5
  Git commit:       bc3805a
  Built:            Thu Feb  9 19:46:49 2023
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.16
  GitCommit:        31aa4358a36870b21a992d3ad2bef29e1d693bec
 runsc:
  Version:          release-20230130.0
  GitCommit:        
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

### runsc debug logs (if available)

```shell
W0215 03:20:30.967056     649 util.go:49] FATAL ERROR: creating loader: converting capabilities: unknown capability "CAP_PERFMON"
creating loader: converting capabilities: unknown capability "CAP_PERFMON"
```

**Comment by oliverchang** [comment]
Docker 20.10.23 appears to be the first version that causes this issue. 20.10.22 still works.

**Comment by nlacasse** [comment]
I think I have a fix in https://github.com/google/gvisor/pull/8530

Can you try patching that to confirm it works?

**Comment by nlacasse** [comment]
Ah hold on.  That PR is not quite right yet.

**Comment by nlacasse** [comment]
OK I fixed up PR #8530, and it should be working now.

I will submit soon.

**Comment by oliverchang** [comment]
> OK I fixed up PR #8530, and it should be working now.
> 
> I will submit soon.

Thanks a ton @nlacasse !
